src/pages/VaultList/components/Vault/Vault_View.tsx

```diff
@@ -64,35 +65,35 @@ const VaultView = ({
                       <div className="flex flex-col">
                         <button
                           className={
-                            "px-5 py-1 hover:bg-gray-100 rounded-md border border-gray-500 shadow-md" +
+                            "px-5 py-3 hover:bg-gray-100 rounded-md border border-[#5a51e5] shadow-md" +
```

Comment:
do we have a tailwind custom color for `#5a51e5`? if so can we add it as i can see its being used multiple times